build/build.cake

```diff
@@ -18,7 +18,7 @@ var configuration = Argument("configuration", "Release");
 // VERSIONS
 //////////////////////////////////////////////////////////////////////
 
-var gitVersioningVersion = "2.1.65";
+var gitVersioningVersion = "2.3.138";
 var inheritDocVersion = "2.0.2";
```

Comment:
Seeing as you're not using InheritDoc any more, remove all the references from here.